livekit-agents/livekit/agents/worker.py

```diff
@@ -313,9 +314,11 @@ async def _worker_task(self) -> None:
                     break
 
                 if retry_count >= self._opts.max_retry:
-                    raise Exception(
-                        f"failed to connect to livekit after {retry_count} attempts: {e}"
+                    logger.error(
```

Comment:
we don't want to throw?

Reply:
the reason is that the error were showing twice, because the cli also log exception on worker.run()

Reply:
I'll double check to see if it introduced an issue where some code was waiting for an exc

Reply:
yeah you're right, reverting to an exception